test/instrumentation/drop-fast-exit-spans.test.js

```diff
@@ -71,7 +71,7 @@ tape.test('end to end test', function (t) {
 // Test that a composite span faster than `exitSpanMinDuration` is dropped.
 tape.test('end to end test with compression', function (t) {
   resetAgent(function (data) {
-    t.equals(data.spans.length, 0, 'the composite span was dropped')
+    t.equals(data.spans.length, 0, `the composite span was dropped (exitSpanMinDuration=${agent._conf.exitSpanMinDuration * 1000}ms, data.spans=${JSON.stringify(data.spans)})`)
```

Comment:
Just curious about this change: is what you are printing out debugging info in case the composite span wasn't dropped?

Reply:
Yup, exactly that. I'd had two spurious failures of that test in CI test runs for this PR, and then, of course, the test passed after I added this change to help debug it.